test/e2e/fixture-builder.js

```diff
@@ -1495,6 +1495,141 @@ class FixtureBuilder {
     return this.withNameController({ names: {} });
   }
 
+  withLedgerAccount() {
```

Comment:
Is there a reson for specifing all these controller-specific state instead of letting the controllers generate their state based on what comes out of KeyringController's one? It feels like a lot of code to be kept updated on each state type change, while I'm not sure we get real benefits in terms of test assertions

Reply:
`PreferencesController` should also be capable of deriving its state from `KeyringController`'s, so we can probably remove `withPreferencesController` as well